src/main/java/meditracker/MediTracker.java

```diff
@@ -76,7 +88,7 @@ public void run() {
      * It creates a new MediTracker object and calls its run() method.
      * @param args Command-line arguments.
```

Comment:
Please add the relevant javadoc for the exception thrown

Reply:
Thx for the review, added in c7699f5